### PR TITLE
Prevent some useless generations

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
However before merging, I'd like to be able to confirm that it works (on GObject) but I wasn't able to generate the GObject files for some reason... Am I missing something?

### CODE REVIEW

File: src/codegen/enums.rs [hunk 1]
```diff
@@ -57,6 +57,10 @@ pub fn generate(env: &Env, root_path: &Path, mod_rs: &mut Vec<String>) {
     }
     imports.add("glib::translate::*", None);
 
+    if !has_any {
```
Comment (EPashkin) [inline src/codegen/enums.rs]: `if has_any` for "mod enums;" don't need then.

Comment (EPashkin) [reply src/codegen/enums.rs]: in flags `let mut first = true` unneeded to for "mod flags;"

File: src/codegen/enums.rs [hunk 2]
```diff
@@ -57,6 +57,10 @@ pub fn generate(env: &Env, root_path: &Path, mod_rs: &mut Vec<String>) {
     }
     imports.add("glib::translate::*", None);
```
Comment (sdroege) [inline src/codegen/enums.rs]: Should probably also not any imports when not generating this file anyway

Comment (EPashkin) [reply src/codegen/enums.rs]: You right, `if !has_any {return}` need be sooner

**Comment by GuillaumeGomez** [comment]
Updated! Once merged, I'll send the update for glib.

**Comment by EPashkin** [comment]
@GuillaumeGomez Thanks, 👍

**Comment by GuillaumeGomez** [comment]
@EPashkin Should we merge it then?

**Comment by EPashkin** [comment]
Yes

**Pull request merged**